ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,176 @@
+---
+eip: xxxx
```

Comment:
```suggestion
eip: 7654
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.